docs/DesignDocs/SGX_Endorsements_V2.md

```diff
@@ -0,0 +1,254 @@
+SGX Attestation Endorsements V2 Updates
+================
+
+Intel SGX released v2 of the Provision Certification Service for ECDSA Attestation.  The
+following APIs were affected:
+
+1. Get TCB Info.
+2. Get Quoting Enclave Identity.
+
+[API Version 1](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate)
+
+[API Version 2](https://api.portal.trustedservices.intel.com/documentation#pcs-certificate-v2)
+
+This document describes the changes for supporting version 2 of the API as well as keeping
+the existing support for version 1 of the API.  This document assumes the user is familiar with
+the SGX attestation endorsements and how it is used during SGX quote verification.
+
+Motivation
+----------
+
+This change is required for customers who would like to use the v2 API web endpoints. Note that the Azure PCK
```

Comment:
This paragraph is azure specific but line 1 is not Azure specific.  If the entire document is Azure-only and not on-prem then please update line 1.  However, I suspect part of this doc are not azure-only, and so maybe it's better to prefix such sentences as this with conditions like "If SGX is being used in an Azure data center context, ..."

Reply:
Resolved.  The changes should apply to either non-Azure DCAP clients as well.

Reply:
Agree, change looks good, thanks.